dictionary/README.md

```diff
@@ -2,4 +2,8 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
```

Comment:
# -2

I thinkk :permanent is a better strategt here. In theory, the dictionary will never exit normally, so transient and permanent would have the same effect. However, :permanent better expresses our intention: this should always be running.